securedrop/journalist_templates/edit_account.html

```diff
@@ -4,14 +4,30 @@
 {% if user %}
   {# Only admins may edit usernames and admin status #}
   <h1>{{ gettext('Edit user "{user}"').format(user=user.username) }}</h1>
-  <p><a href="/admin">« {{ gettext('Back to admin interface') }}</a></p>
-  <h2>{{ gettext('Change Username &amp; Admin Status') }}</h2>
+  <p><a href="/admin">{{ gettext('Back to admin interface') }}</a></p>
+  <h2>{{ gettext('Change Name &amp; Admin Status') }}</h2>
   <form method="post">
     <input name="csrf_token" type="hidden" value="{{ csrf_token() }}">
     <p>
-      <label for="username">{{ gettext('Change username') }}</label>
+      <label for="username">{{ gettext('Username') }}</label>
       <input name="username" id="username" type="text" placeholder="{{ user.username }}">
     </p>
+    <p>
+      <label for="first_name">{{ gettext('First name') }}</label>
+      {% if not user.first_name %}
```

Comment:
Nit: I think it's more readable to just do `user.first_name or ''` in the one place this is used. If not, I think doing this is a little more readable.

```jinja
{% set first_name = user.first_name or '' %}
```

Reply:
Also these checks exist in two places and we may want to DRY it some. Though this may not be possible.